app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -531,7 +531,7 @@
     {{/ui-dropdown}}
   </div>
   <br>
-  <div class="{{unless device.isMobile 'large right floated'}} ui buttons">
+  <div class="{{unless device.isMobile 'right floated' 'three'}} ui buttons large">
```

Comment:
@utkarshnath the `large` class was removed for mobile for the same reasons as pointed in the comment by @sumedh123 , please place it back there.